File: templates/base/flask/src/routes/health.py
import time

from flask import Blueprint, jsonify

health_bp = Blueprint("health", __name__)

_start_time = time.time()


@health_bp.route("/health")
def health():
    return jsonify({
        "status": "ok",
        "uptime": round(time.time() - _start_time, 2),
        "timestamp": int(time.time()),
    })

File: templates/base/elysiajs/src/routes/health.ts
import { Elysia } from "elysia";

const startTime = Date.now();

const healthRoutes = new Elysia()
  .get("/health", () => ({
    status: "ok",
    uptime: Math.floor((Date.now() - startTime) / 1000),
    timestamp: new Date().toISOString(),
  }));

export { healthRoutes };
